doc/whats_new.rst

```diff
@@ -156,7 +156,11 @@ Enhancements
 
 Bug fixes
 .........
-   - Fixed a bug where :class:`sklearn.cluster.DBSCAN` gives incorrect 
+   - Fix a bug regarding fitting :class:`sklearn.ensemble.IsolationForest` with
+     an incorrect formula for the _average_path_length method
```

Comment:
Let's not refer to a private method. Say something "Fixed a bug where IsolationForest used an incorrect calculation of average path length."